### ISSUE TITLE
Checkpoint

### ISSUE THREAD

**Issue by bitwangyujia** [tlkh/text-emotion-classification#9]
When I run the code, it appreas to have no checkpoints of "0.91.h5", how can I solve this problem? I am looking forward for your reply. Thanks a lot.

**Comment by ArsalanRaza** [comment]
This is how i understood it.
comment this line and run the training epochs. Once you have trained models or checkpoints, replace the commented checkpoint_name with recently trained checkpoint and run training again. 
Only this time, the accuracy and validation accuracy will start from where the last training epoch ended.
"It improves the precision of trained model"
